prometheus-formula/prometheus/init.sls

```diff
@@ -25,6 +26,8 @@ config_file:
     - require:
       - pkg: install_prometheus
       - pkg: install_alertmanager
+    - defaults:
+        uyuni_server_hostname: {{ uyuni_server_hostname }}
```

Comment:
I am not sure if this could be an issue, but here (and below) you are using 4 spaces of indentation, while otherwise we are using only 2.

Reply:
Good catch. It worked as expected but it's better to have it consistent.